tooling/api/src/window.ts

```diff
@@ -1157,13 +1157,24 @@ class WebviewWindow extends WindowManager {
 }
 
 /** The WebviewWindow for the current window. */
-const appWindow = new WebviewWindow(
-  window.__TAURI_METADATA__.__currentWindow.label,
-  {
+let appWindow
+if ('__TAURI_METADATA__' in window) {
+  appWindow = new WebviewWindow(
+    window.__TAURI_METADATA__.__currentWindow.label,
+    {
+      // @ts-expect-error
+      skip: true
+    }
+  )
+} else {
+  console.error(
+    'Could not find `window.__TAURI_METADATA__`. The `appWindow` value will reference the `main` window label.'
+  )
```

Comment:
Is this really an error though? It seems it could be a warning.

Also, it could be nice to be able to disable this kind of warnings with an option, an [`import.meta`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import.meta) attribute or in the `window`, like `window.__TAURI_WARNING__ = false` or something.
Running an web app in a web browser seems like a rather legit use case.

Reply:
I agree that an error is a bit much. We should change it to a `console.warn`.
Let's not add a special window property to disable errors, this constant will be deprecated and gone as soon as possible anyway.
But @lucasfernog maybe we can make the message more explicit and mention that it's not necessarily cause for concern and the warning will be removed in a future version

Reply:
done